src/mediapipe_internal/mediapipegraphexecutor.cpp

```diff
@@ -738,13 +757,23 @@ static Status deserializeTensor(const std::string& requestedName, const KFSReque
             case ov::element::Type_t::boolean: {
                 COPY_INPUT_VALUE_BY_VALUE(bool, bool);
             }
+            case ov::element::Type_t::string: {
+                uint32_t offset = 0;
+                for (auto contents : request.inputs(inputIndex).contents().bytes_contents()) {
+                    uint32_t size = contents.size();
```

Comment:
This is used only below as a buffer to copy from. Do we need that? If so, can we make it const and replace following `contents.size()` calls?

Reply:
Can it be const?